llama_index/vector_stores/google/generativeai/genai_extension.py

```diff
@@ -0,0 +1,534 @@
+"""Temporary high-level library of the Google GenerativeAI API.
```

Comment:
Just so I understand, we'll do check this in until the python client lib is ready?